android-lib/src/main/java/com/kinvey/android/Client.java

```diff
@@ -431,7 +431,7 @@ public Context getContext(){
      * <p/>
      * This Builder class is not thread-safe.
      */
-    public static class Builder extends AbstractClient.Builder {
+    public static class Builder<T extends User> extends AbstractClient.Builder {
```

Comment:
Will this be a breaking change to the client builder?

Reply:
No, I won't be breaking change.